### PR TITLE
aliens break cuffs instantly

### PR DESCRIPTION
**Pull request by hornygranny**
cuff_resist for carbons can be called with cuff_break = 2 as an argument, which will allow the mob to instantly break cuffs

### CODE REVIEW

**Comment by Ultimate-Chimera** [comment]
:+1:

**Comment by optimumtact** [comment]
MODS

File: code/modules/mob/living/carbon/carbon.dm [hunk 1]
```diff
@@ -407,7 +407,7 @@
 		else
 			src << "<span class='warning'>You fail to remove [I]!</span>"
 
-	else
+	else if(cuff_break == 1)
```
Comment (RemieRichards) [inline code/modules/mob/living/carbon/carbon.dm]: ```#defines``` pls.

**Comment by PKPenguin321** [comment]
you need a fix pr to balance this

**Comment by hornygranny** [comment]
@PKPenguin321 im working on it

**Comment by KorPhaeron** [comment]
Reminder that under our NEW RULES you will have to fix a bug before you can merge this PR

**Comment by KorPhaeron** [comment]
@WJohn 

Your call.

Keep in mind its pretty fucking obnoxious to be hit by PK borg bolas

**Comment by Anonmare** [comment]
Bear in mind that bucklecuffed xenos otherwise take as long as a Human to break out. Which is kind of hilarious but let's not encourage Wjohnston :^^^^^^^)

**Comment by WJohn** [comment]
:+1:

File: code/modules/mob/living/carbon/carbon.dm [hunk 2]
```diff
@@ -430,6 +430,23 @@
...
+
+		if(I == handcuffed)
+			handcuffed = null
+			update_handcuffed()
+			return
+		else if(I == legcuffed)
+			legcuffed = null
+			update_inv_legcuffed()
+			return
+		return 1
```
Comment (phil235) [inline code/modules/mob/living/carbon/carbon.dm]: can you try to find a way to reduce the copypasta?

Comment (optimumtact) [inline code/modules/mob/living/carbon/carbon.dm]: also return TRUE

**Pull request merged**